build_tools/github_actions/build_dist.py

```diff
@@ -112,6 +113,11 @@ def install_python_requirements():
       [sys.executable, "-m", "pip", "install", "-r", BUILD_REQUIREMENTS_TXT])
 
 
+def configure_bazel():
+  print("Generating configured.bazelrc...")
+  subprocess.check_call([sys.executable, CONFIGURE_BAZEL_PY])
```

Comment:
I think the reason I did this the other way before (adding the call to the script) was because this actually edits the repository and the instructions for running locally have you symlink your repository. Not a huge deal

Reply:
Oh also `check_call` is the old subprocess API. Please use `subprocess.run` (See https://docs.python.org/3/library/subprocess.html)

Reply:
Yeah, I missed the nuance in review, but it has to be this way. In some flows, this is the only script that gets run in both the right container and with the right versions of software (i.e. python, etc) setup in the environment. The way you had it was configuring on the host but not in the container with the right version of python selected (which happens once for each version of Python).

Reply:
I think this is a fair limitation. Hopefully configure_bazel should be idempotent, so it's safe to run again and also hopefully anyone messing with the release build understands enough about this configuration stuff to not be caught off guard.